constructor/header.sh

```diff
@@ -556,20 +578,13 @@ if [ "$TEST" = "1" ]; then
     printf "INFO: Running package tests in a subshell\\n"
     (. "$PREFIX"/bin/activate
      which conda-build > /dev/null 2>&1 || conda install -y conda-build
-#if keep_pkgs
      if [ ! -d "$PREFIX"/conda-bld/__PLAT__ ]; then
          mkdir -p "$PREFIX"/conda-bld/__PLAT__
      fi
      cp -f "$PREFIX"/pkgs/*.tar.bz2 "$PREFIX"/conda-bld/__PLAT__/
      cp -f "$PREFIX"/pkgs/*.conda "$PREFIX"/conda-bld/__PLAT__/
      conda index "$PREFIX"/conda-bld/__PLAT__/
      conda-build --override-channels --channel local --test --keep-going "$PREFIX"/conda-bld/__PLAT__/*.tar.bz2
-#endif
-#if not keep_pkgs
-#          conda-build -t -k "$PREFIX"/pkgs/*
-     printf "ERROR: Testing not possible without keep_pkgs\\n"
```

Comment:
There is no need for this error anymore because we can keep the package cache in place for testing if that option is selected. We still clear the package cache after tests are complete.

Reply:
Or more precisely, after the packages have been staged in the `conda-bld/__PLAT__` subdirectory where the testing occurs.